src/Chapter2.hs

```diff
@@ -741,8 +774,12 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [Int]
```

Comment:
Here, in the result, it is expected to have all of the list that contain the given element, so the type of the function will look like:

```suggestion
contains :: Int -> [[Int]] -> [[Int]]
```

Reply:
wow.. I didn't realize when I built the function. I managed to fix